### PR TITLE
mgr/dashboard: add Services e2e tests

### PR DESCRIPTION
**Pull request by avanthakkar**
Adding e2e specs for creation and deletion of rgw & ingress(supporting this [functionality](https://github.com/ceph/ceph/pull/40936)) service types.

Fixes: https://tracker.ceph.com/issues/50567
Signed-off-by: Avan Thakkar <athakkar@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/05-services.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    services.navigateTo();
+  });
+
+  describe('when Orchestrator is available', () => {
+    // beforeEach(function () {
+    //   cy.fixture('orchestrator/inventory.json').as('hosts');
+    //   cy.fixture('orchestrator/services.json').as('services');
+    // });
+
+    it('should create a RGW service', function () {
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/05-services.e2e-spec.ts:18]: To be aligned with others:
```suggestion
    it('should create a RGW service', () => {
```

**Comment by avanthakkar** [comment]
Currently the delete e2e is failing because it takes some time to delete a service, so I'm exploring on the ways to pass that tests. 
``` 
Services page
    when Orchestrator is available
      ✓ should create a RGW service (6245ms)
      ✓ should create a ingress service (5493ms)
      1) should delete an existing service


  2 passing (25s)
  1 failing

  1) Services page
       when Orchestrator is available
         should delete an existing service:
     UnsubscriptionError: The following error originated from your application code, not from Cypress.

  > 1 errors occurred during unsubscription:
1) TypeError: Cannot read property 'refreshBtn' of undefined
```

**Comment by avanthakkar** [comment]
jenkins retest this please

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html [hunk 2]
```diff
@@ -14,6 +14,7 @@
                  i18n>Type</label>
           <div class="cd-col-form-input">
             <select id="service_type"
+                    name="service_type"
```
Comment (avanthakkar) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html:17]: Change reason: As `selectOption` in e2e  file requires `select[name=` to serach for a service name.

**Review by alfonsomthd — APPROVED** [review]

**Comment by avanthakkar** [comment]
jenkins test dashboard

**Pull request merged**